website/docs/cloud-docs/workspaces/dynamic-provider-credentials/hcp-configuration.mdx

```diff
@@ -0,0 +1,110 @@
+| `TFC_HCP_WORKLOAD_IDENTITY_AUDIENCE`          | Will be used as the `aud` claim for the identity token. Defaults to the provider resource name for the current run phase that was derived from the values you provided for `RUN_PROVIDER_RESOURCE_NAME`, `PLAN_PROVIDER_RESOURCE_NAME`, and `APPLY_PROVIDER_RESOURCE_NAME` | Requires **v1.15.1** or later if self-managing agents. This is one of the default `aud` formats that HCP accepts.                          |
+| `TFC_HCP_PLAN_PROVIDER_RESOURCE_NAME`         | The resource name of the workload identity provider that will be used to assume the service principal for the plan phase of a run.                                                                                                                                         | Requires **v1.15.1** or later if self-managing agents. Will fall back to the value of `RUN_PROVIDER_RESOURCE_NAME` if not provided.        |
+| `TFC_HCP_APPLY_PROVIDER_RESOURCE_NAME`        | The resource name of the workload identity provider that will be used to assume the service principal for the apply phase of a run.                                                                                                                                        | Requires **v1.15.1** or later if self-managing agents. Will fall back to the value of `RUN_PROVIDER_RESOURCE_NAME` if not provided.        |
+
+## Configure the HCP Provider
+
+Make sure that you’re not setting values for the `HCP_CRED_FILE`environment variables as these will conflict with the dynamic credentials authentication process.
+
+### Specifying Multiple Configurations
+
+~> **Important:** If you are self-hosting [Terraform Cloud agents](/terraform/cloud-docs/agents), ensure your agents use [v1.15.1](/terraform/cloud-docs/agents/changelog#1-15-1-05-01-2024) or above. To use the latest dynamic credentials features, [upgrade your agents to the latest version](/terraform/cloud-docs/agents/changelog).
+
+You can add additional configurations to handle multiple distinct HCP setups, enabling you to use multiple [provider aliases](/terraform/language/providers/configuration#alias-multiple-provider-configurations) within the same workspace.
+
+For more details, see [Specifying Multiple Configurations](/terraform/cloud-docs/workspaces/dynamic-provider-credentials/specifying-multiple-configurations).
+
+#### Required Terraform Variable
+
+To use additional configurations, add the following code to your Terraform configuration. This lets HCP Terraform supply variable values that you can then use to map authentication and configuration details to the correct provider blocks.
+
+```hcl
+variable "tfc_hcp_dynamic_credentials" {
+  description = "Object containing HCP dynamic credentials configuration"
+  type = object({
+    default = object({
+      credential_file = string
+    })
+    aliases = map(object({
+      credential_file = string
+    }))
+  })
+}
+```
+
+#### Example Usage
+
```

Comment:
Do you think it'd be helpful to link the [example Terraform configuration](https://github.com/hashicorp/terraform-dynamic-credentials-setup-examples/tree/main/hcp) down here too?

Reply:
I think linking it once should be sufficient, and the context for this block here is talking about how to set up dynamic credentials with aliased providers, whereas the example terraform configuration is the most basic/user-friendly example we could create.

Reply:
Sounds good! Thank you for that extra context!